app/components/seeder-block.js

```diff
@@ -29,8 +24,6 @@ export default Component.extend({
 
         generateAction() {
             if (this.get('isCounterValid')) {
-
-                // Action up to Seeder Controller with the requested amount
```

Comment:
FYI  
There is no problem in having comments that improve code readability. This is very different from having unused, commented code

Reply:
I should asked you before about this. Was an issue for me and sounded really strange no comments at all in the code since the ones that improve code readability are very usefull. Is it better I do it again in another branch or just proceed to the next issue? I think it´s worthwhile because there is a lot of comments that I erased are usefull. What do you think?

Reply:
@tpetrone It´s very fast to do so I already created another branch in my computer and made the cleaning without erasing usefull comments. Can I close this pull request and open another with the new branch for this issue?

Reply:
@kikollovet in these cases, you can simply create another PR that points to the previous branch you were working on. Ideally, You should have had derived from it too, but since there were no changes in `master` branch since then, you should be good.  

In other words, PRs not necessarily points to `master`